pints/noise.py

```diff
@@ -230,3 +230,48 @@ def arma11_unity(rho, theta, sigma, n):
     for i in range(1, n):
         e[i] = (1 - rho) + rho * e[i - 1] + v[i] + theta * v[i - 1]
     return e
+
+
+def multiplicative_gaussian(eta, sigma, f):
```

Comment:
Does this work for multi-output data?

Reply:
Currently it doesn't - I edited the docstring to explicitly state that, and also check that `f` is a single time-series. For now, you would have to call the function separately for each output dimension. However, it wouldn't be too hard to generalize this function to handle multi output data in one call. For multi output data, I think we would want to allow a different value of `eta` and `sigma` for each output (the multiplicative likelihood certainly assumes that they can be different for each output).

Let me know if you think this would be a useful and worthwhile feature.